packages/clerk-auth/src/index.test.ts

```diff
@@ -1,36 +1,31 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
 import { Hono } from 'hono'
-import { clerkMiddleware, getAuth } from '../src'
+import { clerkMiddleware, getAuth } from '.'
 
 const EnvVariables = {
   CLERK_SECRET_KEY: 'TEST_API_KEY',
   CLERK_PUBLISHABLE_KEY: 'TEST_API_KEY',
 }
 
-const authenticateRequestMock = jest.fn()
+const authenticateRequestMock = vi.fn()
+
+vi.mock(import('@clerk/backend'), async (importOriginal) => {
```

Comment:
`vi.mock` also supports types, so I changed the implementation of the mocks slightly to include the original properties, along with the mocked properties